app/src/sharedTest/java/org/oppia/android/app/devoptions/DeveloperOptionsActivityTest.kt

```diff
@@ -237,6 +243,18 @@ class DeveloperOptionsActivityTest {
     }
   }
 
+  @Test
+  fun testDeveloperOptionsFragment_clickEventLogs_opensViewEventLogsActivity() {
```

Comment:
One suggestion, we should separate the tests into `DeveloperOptionsActivityTest` and `DeveloperOptionsFragmentTest`.

Reply:
As per discussion, I have added the TODO and filed an issue for the same #3418. PTAL. Thanks.